x-pack/legacy/plugins/integrations_manager/public/screens/detail/screenshots.tsx

```diff
@@ -19,15 +19,19 @@ export function Screenshots(props: ScreenshotProps) {
   const { images } = props;
 
   // for now, just get first image
-  const src = toImage(images[0].src);
+  const image = images[0];
+  const hasCaption: boolean = images[0].title ? true : false;
 
   const horizontalPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 2;
```

Comment:
I don't think we gain anything from the explicit `boolean` and `number` typing in this file. In those three lines TS can easily determine the type. We're returning `true`/`false` in one and using `parseInt` & a math operation in the other two.

My preference is to use them either when a) we want to make the "contract" very clear (e.g. API response type, component property, etc) b) TS can't infer the value on its own.